webcompat/form.py

```diff
@@ -380,6 +382,18 @@ def domain_name(url):
     return domain
 
 
+def switch_problem_subtype(type):
+    """Return the correct problem subtype object in base of the selected
+    problem type.
```

Comment:
nit: function doc first line should be on one line only.

```
"""Return the correct problem subtype object in base of the selected problem type."""
```

Reply:
As I wrote above, this function needs to be removed, which eliminates the need for this update.